x/market/keeper/deal.go

```diff
@@ -209,3 +231,19 @@ func SetCurNumData(deal types.Deal) {
 	curNumData := deal.GetCurNumData() + 1
 	deal.CurNumData = curNumData
 }
+
+func (k Keeper) Verify(ctx sdk.Context, validatorAddr sdk.AccAddress, unSignedCert types.UnsignedDataValidationCertificate, cert types.DataValidationCertificate) (bool, error) {
```

Comment:
I think you forgot about this comment: https://github.com/medibloc/panacea-core/pull/256#discussion_r788489685  :)